sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/json/JacksonGenerator.scala

```diff
@@ -119,10 +119,6 @@ private[sql] class JacksonGenerator(
       (row: SpecializedGetters, ordinal: Int) =>
         gen.writeNumber(row.getDouble(ordinal))
 
-    case CalendarIntervalType =>
```

Comment:
This doesn't change behavior, right? same string comes out?

Reply:
Yup, as it will still call `toString`.